package.json

```diff
@@ -10,34 +10,32 @@
     },
     "scripts": {
         "analyze": "lit-analyzer \"{packages,tools}/*/src/**/!(*.css).ts\"",
+        "analyze:docs": "cem analyze --globs \"packages/**/*.ts\" --exclude \"**/*.d.ts\" --exclude \"**/stories/**\" --exclude \"**/icons/**\" --exclude \"**/elements/**\" --outdir projects/documentation --litelement",
+        "analyze:storybook": "wireit",
         "build": "wireit",
-        "build:clear-cache": "rimraf packages/*/tsconfig.tsbuildinfo && rimraf tools/*/tsconfig.tsbuildinfo",
-        "build:component-inventory": "node ./tasks/build-component-inventory.js",
+        "build:clean": "rimraf packages/*/tsconfig.tsbuildinfo tools/*/tsconfig.tsbuildinfo",
```

Comment:
I migrated `build:clear-cache` to `build:clean`. Clear-cache wasn't being used anywhere else but it seemed right to keep it as an available shorthand script.

Reply:
nit: since this is related to `ts` what about `build:ts:clean`? am I overthinking it?